Automattic/wp-calypso
# AI Logo generator: add data store for AI Assistant data

Related to https://github.com/Automattic/jetpack/issues/34721

## Proposed Changes

* add store to manage AI assistant data
* add new siteId prop to the component

## Testing Instructions
* Visit calypso home on a site (`/home/[site]`)
* Find the Create logo with AI among the quick links (likely, last one)
* Click and open the modal.

- You should see the modal populated with all the AI Assistant data wrapped in a `<pre>`
- At the top, there should be a primary button labeled "Increase". Clicking it should increase the requests count, noticeable on the `<pre>`.

## Pre-merge Checklist
